openfast_toolbox/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -1659,8 +1662,15 @@ def TS_low_setup(self, writeFiles=True, runOnce=False):
                 # By passing low_ext, manual mode for the domain size is activated, and by passing ds_low,
                 # manual mode for discretization (and further domain size) is also activated
                 currentTS = TSCaseCreation(D_, HubHt_, Vhub_, tivalue_, shear_, x=xlocs_, y=ylocs_, zbot=self.zbot, cmax=self.cmax,
-                                           fmax=self.fmax, Cmeander=self.Cmeander, boxType=boxType, low_ext=self.extent_low, ds_low=self.ds_low_les)
+                                           fmax=self.fmax, Cmeander=self.Cmeander, boxType=boxType, low_ext=self.extent_low, ds_low=self.ds_low)
                 self.TSlowbox = currentTS
+
+                # Ensure that dt low is a multiple of dt high and that tmax is a multiple of both dt low and dt high, and that tmax is larger than tmax_desired
+                currentTS.dt = getMultipleOf(currentTS.dt, multipleof=self.dt_high)
+                self.tmax = getMultipleOf(self.tmax_desired, multipleof=currentTS.dt)
+                if self.tmax < self.tmax_desired:
+                    self.tmax += currentTS.dt
```

Comment:
Can you give me an example of when this is needed?

Reply:
this ensures that low-res and high-res turbsim files are compatible. if we don't enforce this, we have weird mismatches when creating the time series exatracted from the low-res and passed to the high res. for example, your code was setting max at 900s and dt at 4.1xxx. but those aren't multiple, so turbsim was actually creating a low res box ending at 901.xxx. we then extracted the time series, which however had to have the time step of the high res. I added the interpolation, which was missing (because dt in reality was hard coded to be the same between low res and high res, and this PR fixes that), but then again at tmax things were a little funky. conclusion: I believe it's cleaner if we set multiples